packages/osintgram/PKGBUILD

```diff
@@ -0,0 +1,49 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osintgram
+pkgver=348.50d44fd
+pkgrel=1
+pkgdesc='OSINT tool on Instagram. It offers an interactive shell to perform analysis on Instagram account of any users by its nickname.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/Datalux/Osintgram'
+license=('GPL3')
+depends=()
```

Comment:
You forgot to add runtime dependencies.

https://github.com/Datalux/Osintgram/blob/master/requirements.txt

Reply:
@noraj should they not be managed by `pip install -r requirements.txt` in `prepare()`?

Reply:
Only for virtualenv installs that we absolutely try to avoid. Here we don't need a venv nor a `prepare()`. As the next comment state list all the dependencies that are already existing here and package the missing oens (`instagram-private-api`).

Reply:
Ok thanks @noraj . Currently I retrieved the `instagram-private-api` source file from https://pypi.org/project/instagram-private-api/#files
I need only to understand how to manage it in this package. I will try to do it tomorrow.

Reply:
All you have to do in this case for this lib is filling https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools. It should be very easy as there is no deps https://github.com/ping/instagram_private_api/blob/0fda0369ac02bc03d56f5f3f99bc9de2cc25ffaa/setup.py#L34

Reply:
Thanks. A theorical question: can you provide me the reason why virtualenv installs should be avoided? What are the concerns?

Reply:
> All you have to do in this case for this lib is filling https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools. It should be very easy as there is no deps https://github.com/ping/instagram_private_api/blob/0fda0369ac02bc03d56f5f3f99bc9de2cc25ffaa/setup.py#L34

Do you mean that we need to create a PKGBUILD, so a BA package, for `instagram-private-api`, right? If so, in which folder I should submit it? I don't think in `BlackArch/blackarch/packages` folder.

Reply:
PS: created PR for `python-instagram-private-api`: https://github.com/BlackArch/blackarch/pull/3657

Reply:
You know when you edit a file with github webui to prepare your PR it basically just create a branch `patch-X` on your fork (eg. https://github.com/D3vil0p3r/blackarch/tree/patch-8 here) based on [BlackArch:master](https://github.com/BlackArch/blackarch). So for example here, you can browse [D3vil0p3r:patch-8](https://github.com/D3vil0p3r/blackarch/tree/patch-8) and create `python-instagram-private-api` PKGBUILD there so it will be added to the same PR rather than creating a new PR for it. 1 PR per tool is nice but when it's the dependencies of the tool it's best to have them in the same PR.

Reply:
Usually, I work by using `git` and on GitHub WebUI I didn't know this stuff. So, if I add something on an already existing fork (like https://github.com/D3vil0p3r/blackarch/tree/patch-8), the proposed PR is updated automatically as well? Ok, I will consider it for the next tasks. Thank you very much.

Reply:
Yeah, the same apply if you work on the CLI.